fuzzers/angora/instrument.sh

```diff
@@ -22,7 +22,7 @@ export CXX="$FUZZER/repo/bin/angora-clang++"
     export LDFLAGS="$LDFLAGS -L$OUT"
 
     export USE_FAST=1
-    export LIBS="$LIBS -l:angora_driver.o"
+    export LIBS="$LIBS -l:angora_driver.o -lstdc++ -lc++"
```

Comment:
Is there a reason for adding `-lc++` here? Mine builds fine without it

Reply:
Only `-lstdc++` is required. Angora had some linking problems when compiling OpenSSL. I must have forgotten to remove `-lc++`.

Reply:
Ok cool. Please remove and update your PR